plugins/dell/fu-plugin-dell.c

```diff
@@ -867,7 +867,7 @@ fu_plugin_init (FuPlugin *plugin)
 	FuPluginData *data = fu_plugin_alloc_data (plugin, sizeof (FuPluginData));
 
 	data->smi_obj = g_malloc0 (sizeof (FuDellSmiObj));
-	if (fu_dell_supported ())
+	if (fu_dell_supported (NULL))
```

Comment:
if it's failable, it's supposed to be called from `setup()`, not `init()`

Reply:
OK will move it over.